pages/blog/blog.module.scss

```diff
@@ -0,0 +1,98 @@
+@import '../../styles/globals.scss';
+
+.container {
+	padding: $space-md $space-xxxl;
+	margin-bottom: $space-xxl;
+
+	p {
+		font-family: $primaryTextFont;
+	}
+
+	header {
+		position: relative;
```

Comment:
lets use `flexbox` for positioning items on the page this way when we add responsive styles the layout will be a little more fluid